btfxwss/queue_processor.py

```diff
@@ -217,8 +194,8 @@ def _handle_auth(self, dtype, data, ts):
         channel_id = data.pop('chanId')
         user_id = data.pop('userId')
 
-        identifier = ('auth', user_id)
-        self.channel_handlers[identifier] = channel_id
+        identifier = ('account', user_id)
```

Comment:
This change from 'auth' to 'account' worked but I do not know for certain that it _did not_ break something obscure. Is this important somewhere else?

Reply:
Nope - It's registered once, and then later on only looked up by channel ID.  So this should be fine.